realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -48,6 +48,18 @@
 public class SyncManager {
 
     /**
+     * Debugging related options.
+     */
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
```

Comment:
Wouldn't it be better to use the BuildConfig class for this? Then we can configure them from Gradle? I'm not sure if we can only inject constants for test build, but I believe it is possible